### ISSUE TITLE
No textures on Plock/Karen

### ISSUE THREAD

**Issue by ajventer** [Poodmund/PoodsOPMVO#3]
As of the latest Beta this is still a problem. All other OPM planets are good, but Plock and Karen have no textures.
![screenshot3](https://cloud.githubusercontent.com/assets/1582463/24618717/4b11f500-1898-11e7-8d1b-0ce600d5e826.png)

**Comment by Poodmund** [comment]
At this time this mod does nothing to those bodies so I suspect it is an issue with OPM and not OPM-VO. 

Could you please test whether this issue persists without the PoodsOPMVO folder in your GameData directory?

I think I may know what the issue is but I want to confirm that it is an issue with OPM and not with my OPM-VO.

Thanks in advance.

EDIT: Also posting your output.txt, KSP.log and ModuleManager.cache files would be handy to further investigate.

**Comment by ajventer** [comment]
Before I added OPM-VO I had no textures on any OPM planets. I thought it was a clash between OPM and SVE/SVT and so I added your mod as a fix. It worked for all but Plock/Karen too. It's entirely possible that it's an OPM/Kopernicus issue - if you could advise me how to fix it I would be foreever in your debt :P

I have no output.txt file.

[KSP.log.txt](https://github.com/Poodmund/PoodsOPMVO/files/890859/KSP.log.txt)
[ModuleManager.ConfigCache.txt](https://github.com/Poodmund/PoodsOPMVO/files/890861/ModuleManager.ConfigCache.txt)

**Comment by Poodmund** [comment]
It is unfortunately an issue with playing on Linux, a lack of DirectX as far as I am aware. Try running the game with the following config patch and see if it helps:
```
@Kopernicus:FINAL
{
	@Body,*
	{
		useOnDemand = False
	}
}
```
A work-around it go through through all the OPM textures and convert them all to .png format; if this is done the filepaths of all the textures in the Kopernicus configs have to be changed to from the .dds extension to .png also.

Let me know how you get on.

**Comment by ajventer** [comment]
I added the config - but it made no difference (I did check the log to confirm it was indeed loaded). So I converted all the textures (run this inside the OPM folder):

 ` for I in `find -iname *.dds` ; do K=`dirname $I`;  J=$K/`basename $I .dds` ; convert $I $J.png ; done
cd KopernicusConfigs
for I in `find | grep .cfg` ; do sed -i "s#.dds#.png#g" $I ; done
`

That fixed it. I'm not sure if it would work without the patch to disable onDemand loading, but I left it in anyway. Thanks for your help, hopefully if somebody else encounters this problem this thread will provide them a fix.
